### PR TITLE
Lab2

### PR DESCRIPTION
**Pull request by Starvotoitov**

### CODE REVIEW

File: Laba2.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<DOCUMENT html>
+<html>
```
Comment (Starvotoitov) [inline Laba2.html:2]: remove wrapper

File: Laba2.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<DOCUMENT html>
+<html>
+<head>
+	<title>������������ 2</title>
```
Comment (Starvotoitov) [inline Laba2.html:6]: Add responisovnes  (media queries )

File: Laba2.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
+<DOCUMENT html>
+<html>
+<head>
+	<title>������������ 2</title>
+	<link rel="stylesheet" href="Laba2.css">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+<body>
+	<div>
```
Comment (Starvotoitov) [inline Laba2.html:11]: semantic html (proper naming for tags)

File: Laba2.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+					<li>UI KITS</li>
+					<li>PREMIUM PSD FILES</li>
+					<li>ABOUT</li>
+					<li>BLOG</li>
+				</ul>
+			</div>
+			<div class="NavigationIcon">
+				<p>Navigation</p>
+				<i class="fas fa-bars"></i>
+				<div style="clear: both; height: 0px;"></div>
```
Comment (Starvotoitov) [inline Laba2.html]: Try to avoid inline styles, use classes instead

File: Laba2.css [hunk 5]
```diff
@@ -0,0 +1,264 @@
+body {
+	font-family: sans-serif;
+}
+
+.NavigationBlock {
```
Comment (Starvotoitov) [inline Laba2.css]: To read: Yandex BEM

**Comment by aliaksandrkrasnou** [comment]
Approved, nice :+1

**Pull request closed without merging**